schema/core.md

```diff
@@ -379,27 +417,14 @@ used for this encoding. For most records, it is anticipated that this will be "e
 ### Creative Work Status
 
 [Schema:creativeWorkStatus](https://schema.org/creativeWorkStatus) is a property of `CreativeWork` used to capture the
-stage of a work's lifecycle. Example terms include Incomplete, Draft, Published, Obsolete, etc. Some organizations define a 
-set of terms for the stages of their publication lifecycle. This can be expressed as text or using the 
-[Schema:DefinedTerm](https://schema.org/DefinedTerm) class.
+stage of a work's lifecycle. Example terms include Incomplete, Draft, Obsolete, and Published. 
```

Comment:
I thought that these were going to be expressed as a controlled list of definedTerms. This would provide more information then a simple text string. For example,

Instead of 
``` json
{
  "creativeWorkStatus": "published"
}
```

We would have
``` json
{
  "creativeWorkStatus": {
    "@type": "DefinedTerm",
    "name": "published",
    "description": "a formally published resource that has been issued a DOI. This resource is immutable"
  }
}
```

Reply:
All terms with their descriptions were added to the schema.